sources/packages/api/src/services/application/application.service.ts

```diff
@@ -817,7 +829,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
         location: { id: locationId },
         pirStatus: Not(ProgramInfoStatus.notRequired),
       },
-      { relations: ["studentFiles"] },
+      { relations: ["studentFiles", "currentAssessment"] },
```

Comment:
Why are we retrieving currentAssessment here? I didn't get that point.

Reply:
The current assessment holds the workflow id and this will be needed to cancel the application being overwritten.
I did not put much effort into refactoring this method because there are some considerations that could lead us to just remove it completely.